ultralytics/utils/callbacks/dvc.py

```diff
@@ -11,15 +11,12 @@
 
     import os
     import re
-    from importlib.metadata import version
     from pathlib import Path
 
-    import pkg_resources as pkg
```

Comment:
@glenn-jocher it got DVC integration broken since we don't use `__version__` (it's [outdated](https://stackoverflow.com/a/72168209/298182) AFAIU). We can add it back in `dvclive` if it's needed, but you clarify the reason for changing this, was there an issue at some point with it?

Reply:
Unrelated (kind of related?) issue: From v8.0.153 -> latest, yolo and dvclive confusion matrix logging also isn't working correctly. From 8.0.154 to 8.0.167 (where this bug occurs), cf.json is empty and confusion matrices aren't showing up in DVC. I tested 8.0.153, and the confusion matrix still works in this verison. 

Just adding here for info so both of you can see. I can create an issue reporting this (or both of these) bug(s) if needed.

Reply:
thanks @BradyJ27 . I can look into both.

Reply:
created the formal issue to scope it and make the discussion more visible

Reply:
@BradyJ27 hmm. I've tested the confusion matrix logging in 8.0.184 and can't reproduce it? can you please create a separate issue for this and give more details on how we can reproduce this please?

Reply:
@shcheklein  I'm unable to reproduce with 8.0.184. It must've been some weird issue with my existing dvclive folder and/or existing experiments. It may have also just been some weird version error since I was using 8.0.16x, sorry for the confusion, thanks for looking into it!